app/models/feed.rb

```diff
@@ -75,6 +77,16 @@ def sanitize_text
     self.title = sanitize_text_only(title) if title_changed?
   end
 
+  def set_public_feeds_url
+    if public? && url.blank? && ENV["PUBLIC_FEEDS_PREFIX"].present?
+      self.url = "#{ENV["PUBLIC_FEEDS_PREFIX"]}/#{podcast.path}/#{published_path}"
+    end
+  end
+
+  def remove_url
+    self.url = nil
+  end
```

Comment:
Fixes a bug where changing a Feed from public -> private would retain the `url`.  Which caused a bad `atom:link` in the RSS.  (The `atom:link` needs to be that `p.prxu.org` for things to work).